languagetool-language-modules/de/src/test/java/org/languagetool/rules/de/GermanSpellerRuleTest.java

```diff
@@ -60,6 +60,7 @@ public class GermanSpellerRuleTest {
   @Test
   public void testIgnoreMisspelledWord() throws IOException {
     GermanSpellerRule rule = new GermanSpellerRule(TestTools.getMessages("de"), GERMAN_DE);
+    assertTrue(rule.ignorePotentiallyMisspelledWord("Weinkühlschrank")); 
```

Comment:
Could you also add negative examples like "Weinskühlschrank", "Weinkühlsschrank" and maybe more to make sure these don't get accepted now?